Client/mods/deathmatch/logic/luadefs/CLuaWorldDefs.cpp

```diff
@@ -2060,3 +2063,174 @@ bool CLuaWorldDefs::ResetCoronaReflectionsEnabled()
     g_pGame->GetSettings()->ResetCoronaReflectionsEnabled();
     return true;
 }
+
+std::variant<bool, float, CLuaMultiReturn<float, float, float>> CLuaWorldDefs::GetWorldProperty(eWorldProperty property)
+{
+    switch (property)
+    {
+        case eWorldProperty::AMBIENT_COLOR:
+        {
+            float red, green, blue;
+            g_pMultiplayer->GetAmbientColor(red, green, blue);
+            return std::make_tuple((int16)(red * 255), (int16)(green * 255), (int16)(blue * 255));
+        }
+        case eWorldProperty::AMBIENT_OBJ_COLOR:
+        {
+            float red, green, blue;
+            g_pMultiplayer->GetAmbientObjectColor(red, green, blue);
+            return std::make_tuple((int16)(red * 255), (int16)(green * 255), (int16)(blue * 255));
+        }
+        case eWorldProperty::DIRECTIONAL_COLOR:
+        {
+            float red, green, blue;
+            g_pMultiplayer->GetDirectionalColor(red, green, blue);
+            return std::make_tuple((int16)(red * 255), (int16)(green * 255), (int16)(blue * 255));
+        }
+        case eWorldProperty::SPRITE_SIZE:
+            return g_pMultiplayer->GetSpriteSize();
+        case eWorldProperty::SPRITE_BRIGHTNESS:
+            return g_pMultiplayer->GetSpriteBrightness();
+        case eWorldProperty::POLE_SHADOW_STRENGTH:
+            return (float)g_pMultiplayer->GetPoleShadowStrength();
+        case eWorldProperty::SHADOW_STRENGTH:
+            return (float)g_pMultiplayer->GetShadowStrength();
```

Comment:
use static_cast instead, pleasee

Reply:
If this is validated, it can be done by addendum.